api_docs/integrations-overview.md

```diff
@@ -1,47 +1,105 @@
+1. [Search Zulip's integrations](/integrations/) for the product you'd like to
+   connect to Zulip.
+
+1. Click on the card for the product, and follow the instructions on the page.
+
+{end_tabs}
+
+### Integrate via Zapier or IFTTT
+
+If you don't see a native Zulip integration, you can access thousands of
+additional integrations through [Zapier](https://zapier.com/apps) and
+[IFTTT](https://ifttt.com/search).
+
+{start_tabs}
+
+1. Search [Zapier](https://zapier.com/apps) or [IFTTT](https://ifttt.com/search)
+for the product you'd like to connect to Zulip.
+
+1. Follow the integration instructions for [Zapier](/integrations/doc/zapier) or
+   [IFTTT](/integrations/doc/ifttt).
+
+{end_tabs}
+
+### Migrate an integration from Slack
+
+Zulip can process incoming webhook messages written to work with [Slack's
+webhook API](https://api.slack.com/messaging/webhooks). This makes it easy to
+quickly move your integrations when [migrating your
+organization](/help/import-from-slack) from Slack to Zulip.
+
+!!! warn ""
+
+     **Note:** In the long term, the recommended approach is to use
+     Zulip's native integrations, which take advantage of Zulip's topics.
+     There may also be some quirks when Slack's formatting system is
+     translated into Zulip's.
+
+{start_tabs}
+
+1. [Create a bot](/help/add-a-bot-or-integration) for the Slack-compatible
+   webhook. Make sure that you select **Incoming webhook** as the **Bot type**.
+
+1. Decide where to send Slack-compatible webhook notifications, and [generate
+   the integration URL](https://zulip.com/help/generate-integration-url).
+
+1. Use the generated URL anywhere you would use a Slack webhook.
+
+{end_tabs}
+
+### Integrate via email
+
+If the product you'd like to integrate can send email notifications, you can
+[send those emails to a Zulip channel](/help/message-a-channel-by-email). The
+email subject will become the Zulip topic, and the email body will become the
+Zulip message.
+
+For example, you can configure your personal GitHub notifications to go to a
+Zulip channel rather than your email inbox. Notifications for each issue or pull
+request will be grouped into a single topic.
 
 ## Write your own integration
 
-We've put a lot of effort into making this as easy as possible, but
-all of the options below do require some comfort writing code. If you
-need an integration and don't have an engineer on staff, [contact
-us](/help/contact-support) and we'll see what we can do.
+You can write your own Zulip integrations using the well-documented APIs below.
+For example, if your company develops software, you can create a custom
+integration to connect your product to Zulip.
+
+If you need help, best-effort community support is available in the [Zulip
+development community](https://zulip.com/development-community/). To inquire
+about options for custom development, [contact Zulip
+Sales](mailto:sales@zulip.com).
 
 ### Sending content into Zulip
 
```

Comment:
I think here we should have one more option here that will often be simplest for folks:

> The `zulip-send` tool makes it easy to send Zulip messages from shell scripts.

It's documented in a tab on https://zulip.com/api/send-message; do we have a way to direct link to it? Maybe regardless, might want to extract it to its own top-level documentation page. Maybe we need a #documentation thread about where to put that, so not a blocker for this PR, I think.